scripts/website.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 
-npx netlify deploy --dir=./dist/website --message="Website - $CIRCLE_BRANCH@$CIRCLE_SHA1"
+curl -g https://us-central1-clarity-design-system.cloudfunctions.net/actions -o .env
```

Comment:
question: is it reasonable to put a short comment describing the purpopse of this  script? (and the other calls in load-vars.sh, dev.sh) 

imo - it would help us as a team to be more familiar with how things get build and deployed.

Reply:
I'll comment a bit, I'm still trying to get it to work.